doc/api.md

```diff
@@ -1,364 +1,675 @@
+```
 
-unindexed biomes
+### `minecraftData.biomesByName`
 
-### minecraft-data.biomesByName
+Biomes indexed by name
 
-biomes object indexed by name
+Example:
+
+```js
+console.log(minecraftData.biomesByName['windswept_gravelly_hills']) // Object containing information for "Windswept Gravelly Hills"
+```
+
+### `minecraftData.biomesArray`
+
+Array of biomes
 
 ## Recipes
 
-### minecraft-data.recipes
+### `minecraftData.recipes`
+
+Recipes indexed by the resulting item id
+
+Example:
 
-recipes indexed by id
+```js
+console.log(minecraftData.recipes[31]) // Recipe information for crafting "Dripstone Block"
+
+// Returns:
+// {
+//   inShape: [ [ 1100, 1100 ], [ 1100, 1100 ] ],
+//   result: { count: 1, id: 13 }
+// }
+
+// Note: 1100 is the block ID of "Pointed Dripstone"
+```
 
 ## Instruments
 
-### minecraft-data.instruments
+### `minecraftData.instruments`
 
-instruments indexed by id
+Instruments indexed by id
 
-### minecraft-data.instrumentsArray
+Example:
 
-unindexed instruments
+```js
+console.log(minecraftData.instruments[5])
+// Returns: { id: 5, name: 'flute' }
+```
+
+### `minecraftData.instrumentsArray`
+
+Array of instruments
 
 ## Materials
 
-### minecraft-data.materials
+### `minecraftData.materials`
+
+Material types indexed by name
 
-materials indexed by name
+Example:
+
+```js
+console.log(minecraftData.materials['mineable/axe'])
+// Returns: { '702': 2, '707': 4, '712': 12, '717': 6, '722': 8, '727': 9 }
+```
 
 ## Entities
 
-### minecraft-data.mobs
+### `minecraftData.mobs`
+
+Mobs (passive, neutral, and hostile) indexed by id
+
+Example:
+
+```js
+console.log(minecraftData.mobs[30]) // Object containing information for "Ghast"
+```
+
+### `minecraftData.objects`
 
-mobs indexed by id
+Objects (non-mob entities such as vehicles and projectiles) indexed by id
 
-### minecraft-data.objects
+Example:
 
-objects indexed by id
+```js
+const minecraftData = MinecraftData('1.8.9')
+console.log(minecraftData.objects[10]) // Object containing information for "Minecart"
+```
+
+### `minecraftData.entities`
+
+Entities indexed by id
+
+Example:
+
+```js
+console.log(minecraftData.entities[25]) // Object containing information for "Evoker"
+```
+
+### `minecraftData.entitiesByName`
 
-### minecraft-data.entitiesByName
+Entities indexed by name
 
-entities indexed by name
+Example:
+
+```js
+console.log(minecraftData.entitiesByName['evoker']) // Object containing information for "Evoker"
+```
 
-### minecraft-data.entitiesArray
+### `minecraftData.entitiesArray`
 
-unindexed entities
+Array of entities
 
 ## Enchantments
 
-### minecraft-data.enchantments
+### `minecraftData.enchantments`
 
-enchantments indexed by id
+Enchantments indexed by id
 
-### minecraft-data.enchantmentsByName
+Example:
 
-enchantments indexed by name
+```js
+console.log(minecraftData.enchantments[37]) // Object containing information for "Mending"
+```
+
+### `minecraftData.enchantmentsByName`
```

Comment:
It looks better without the backtick and more consistent with other places, let's undo this.

Reply:
Done 👍